### PR TITLE
Add pre-shared symmetric key support

### PR DESCRIPTION
**Pull request by LiveChief**

### CODE REVIEW

File: wireguard-install.sh [hunk 1]
```diff
@@ -88,13 +88,10 @@ elif [[ "$OS" = 'arch' ]]; then
     pacman -S wireguard-tools
 fi
 
-# Generate key pair for the server
-SERVER_PRIV_KEY=$(wg genkey)
-SERVER_PUB_KEY=$(echo "$SERVER_PRIV_KEY" | wg pubkey)
-
```
Comment (angristan) [inline wireguard-install.sh]: Why do you remove this?

File: wireguard-install.sh [hunk 2]
```diff
@@ -88,13 +88,10 @@ elif [[ "$OS" = 'arch' ]]; then
...
 # Generate key pair for the server
 CLIENT_PRIV_KEY=$(wg genkey)
 CLIENT_PUB_KEY=$(echo "$CLIENT_PRIV_KEY" | wg pubkey)
+CLIENT_PRE_KEY=$( wg genpsk )
```
Comment (angristan) [inline wireguard-install.sh]: Please make it optional.

File: wireguard-install.sh (hunk 1 above)
Comment (LiveChief) [reply wireguard-install.sh]: this was there 2x times for some reason.

**Comment by lucawen** [comment]
i will make this changes...

**Comment by LiveChief** [comment]
Thanks

**Pull request closed without merging**